wordpress-mobile/WordPress-Android
# Dashboard Personalization: Implement tracking

## Closes 
#18940 

## Description
This PR implements the tracking for click on enabling/disabling a dashboard card on personalize screen 

This PR also renames all the resources related to personalization screen from **personalise** to **personalize** as discussed [here](https://github.com/wordpress-mobile/WordPress-Android/pull/19171#discussion_r1324440633)

## To test:
1. Go to Jetpack app
2. Go to Me → Debug Settings → Enable dashboard_personalization
3. Go to the bottom of dashboard
4. Click on Personalize your home tab
5. Toggle the visibility of any card
6. When the card is enabled ✅ Verify that following event is tracked - `🔵 Tracked: personalisation_screen_card_show_tapped	, Properties: {"type":"todays_stats|pages|activity_log|draft_posts|scheduled_posts|blogging_prompts|next_steps"}`
8. When the card is disabled, verify that the following event is tracked - `🔵 Tracked: personalisation_screen_card_hide_tapped, Properties: {"type":"todays_stats|pages|activity_log|draft_posts|scheduled_posts|blogging_prompts|next_steps"}`


## Merge Instructions
1. Make sure that [Dashboard Personalization: Implements the Personalization Screen](https://github.com/wordpress-mobile/WordPress-Android/pull/19178#issuecomment-1719255163) is merged to trunk
2. Verify that the base branch is updated to trunk
3. Remove - Do not merge - label
4. Merge as Normal

## Regression Notes
1. Potential unintended areas of impact
Personalization screen card state changes are not tracked as expected

5. What I did to test those areas of impact (or what existing automated tests I relied on)
Unit tests

6. What automated tests I added (or what prevented me from doing so)
Unit tests

PR submission checklist:

UI Changes testing checklist:
